Content.Server/Nyanotrasen/Cloning/MetempsychoticMachineSystem.cs

```diff
@@ -0,0 +1,76 @@
+using Content.Shared.Humanoid.Prototypes;
+using Content.Shared.Random;
+using Content.Shared.Random.Helpers;
+using Robust.Shared.Random;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.Cloning
+{
+    public sealed class MetempsychoticMachineSystem : EntitySystem
+    {
+        public const string MetempsychoticHumanoidPool = "MetempsychoticHumanoidPool";
+        public const string MetempsychoticNonHumanoidPool = "MetempsychoticNonhumanoidPool";
```

Comment:
I don't know the names but I believe there's prototype validator attributes you should add to these

Reply:
I genuinely don't know what a prototype validator is or what you're referring to?

Reply:
[ValidatePrototypeId<WhateverPrototype>]

Reply:
Got it